packages/smooth_app/lib/cards/data_cards/image_upload_card.dart

```diff
@@ -46,35 +45,17 @@ class _ImageUploadCardState extends State<ImageUploadCard> {
       if (!mounted) {
         return;
       }
-      final bool isUploaded = await uploadCapturedPicture(
+      await uploadCapturedPicture(
         context,
         barcode: widget.product
             .barcode!, //Probably throws an error, but this is not a big problem when we got a product without a barcode
         imageField: widget.productImageData.imageField,
         imageUri: croppedImageFile.uri,
       );
-      croppedImageFile.delete();
+
       if (!mounted) {
```

Comment:
What the goal of this statement alone now?

Reply:
I deleted that code from that file, 

Though what this code does was When cropping was done, it gives an output to a file in the cache memory, so once we are done with the image upload then we delete the file, so as to not accumulate cache,

Reply:
I think you can remove it, it won't change anything